src/expr/src/vector_op/agg/general_agg.rs

```diff
@@ -256,7 +256,7 @@ mod tests {
                 Arc::new(input.into()),
                 &agg_type,
                 return_type,
-                ArrayBuilderImpl::Int32(I32ArrayBuilder::new(0)?),
+                ArrayBuilderImpl::Int32(I32ArrayBuilder::new(0).unwrap()),
```

Comment:
Why `ArrayBuilder::new` returns `Result`...

Reply:
Allocation may failed 😃

Reply:
Anyway, here it's `unwrap`ed.

Reply:
Here is tests